shapeshift/web
# feat: allow for account selection to send

## Description

Allows user to select an account from which to send bitcoin

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

closes #689 

## Testing

Please outline all testing steps

1. Open send from Bitcoin.
2. Choose an account (segwit, segwit native, legacy)
3. Your transaction should be sent from the account that you chose.

## Screenshots (if applicable)
